language/compiler/ir-to-bytecode/syntax/src/syntax.rs

```diff
@@ -222,11 +241,13 @@ fn get_precedence(token: &Tok) -> u32 {
         Tok::Pipe => 5,
         Tok::Caret => 6,
         Tok::Amp => 7,
-        Tok::Plus => 8,
-        Tok::Minus => 8,
-        Tok::Star => 9,
-        Tok::Slash => 9,
-        Tok::Percent => 9,
+        Tok::LessLess => 8,
```

Comment:
I checked the precedence of these operators in Rust and can confirm that this matches (which is what we have done so far). There's nothing magic about Rust but at least it is consistent with something.